File: Common/Entity/src/main/java/it/ey/dto/Sezione332DTO.java
package it.ey.dto;

import lombok.*;
import lombok.experimental.SuperBuilder;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
public class Sezione332DTO extends SezioneBaseDTO{

    private Long id;

    private Long idPiao;

    private String statoSezione;

    private String contestoNormativo;

    private String descrizioneQualitativa;

    private String descrizioneStrategia;

    private String descrizioneRisorse;

    private String descrizioneIncentivi;

    private List<ObiettiviRisultatiFotografiaDTO> obiettiviRisultatiFotografia;

    private List<AttivitaFormativeDTO> attivitaFormative;

    private List<TabellaFunzionaleDTO> tabelleFunzionali;
}

File: dfp-piao-fe/src/app/shared/models/classes/sezione-332-dto.ts
import { ObiettiviRisultatiFotografiaDTO } from './obiettivi-risultati-fotografia-dto';
import { TabellaFunzionaleDTO } from './tabella-funzionale-dto';
import { AttivitaFormativeDTO } from './attivita-formativa-dto';

export class Sezione332DTO {
  id?: number;
  idPiao?: number;
  statoSezione?: string;
  contestoNormativo?: string;
  descrizioneQualitativa?: string;
  descrizioneStrategia?: string;
  descrizioneRisorse?: string;
  descrizioneIncentivi?: string;

  obiettiviRisultatiFotografia?: ObiettiviRisultatiFotografiaDTO[];

  tabelleFunzionali?: TabellaFunzionaleDTO[];

  attivitaFormative?: AttivitaFormativeDTO[];

  campiModificati?: string;
  testoSezione?: string;
}
